praetorian_cli/sdk/entities/account_discovery.py

```
@@ -0,0 +1,397 @@
+        try:
+            headers = dict(auth_headers)
+            headers['account'] = email
+            resp = requests.get(
+                f'{base_url}/agent/enhanced',
+                headers=headers,
+                timeout=30,
+            )
+            if resp.status_code == 200:
+                return [(Agent.from_dict(d), acct) for d in resp.json()]
+            logger.debug('Agent load for %s returned status %d (attempt %d)', email, resp.status_code, attempt)
+            return None  # Signal failure (distinct from empty account)
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)